QuickRepliesApp.ts

```diff
@@ -91,6 +93,77 @@ export class QuickRepliesApp extends App {
 			blockBuilder: this.blockBuilder,
 		};
 	}
+
+	private getAssociations(userId: string): RocketChatAssociationRecord[] {
+		return [
+			new RocketChatAssociationRecord(
+				RocketChatAssociationModel.USER,
+				userId,
+			),
+			new RocketChatAssociationRecord(
+				RocketChatAssociationModel.MISC,
+				'reply',
+			),
+		];
+	}
+
+	public async onInstall(
```

Comment:
are we sure that this will run only once for each user ? we should run this once for each user who opens the app for the first time make sure we do it that way.
my feel this will run only for admin guy once the app is installed in the server research on this please.